backend/server/src/service/ratings.rs

```diff
@@ -0,0 +1,164 @@
+use crate::models::campaign::Campaign;
+use crate::models::error::ChaosError;
+use crate::models::organisation::{Member, MemberList, OrganisationDetails, OrganisationRole};
+use chrono::{DateTime, Utc};
+use snowflake::SnowflakeIdGenerator;
+use sqlx::{Pool, Postgres, Transaction};
+use std::ops::DerefMut;
+use uuid::Uuid;
+
+/// Any member of the organisation that owns the campaign is an application
+/// viewer, because all members are either directors or execs (TODO: might be
+/// changed in the future).
+pub async fn assert_user_is_application_reviewer_admin_given_application_id(
+    user_id: i64,
+    application_id: i64,
+    pool: &Pool<Postgres>,
+) -> Result<(), ChaosError> {
+    let is_admin = sqlx::query!(
+        r#"
+        SELECT EXISTS (
+            SELECT 1
+            FROM organisation_members om
+            JOIN campaigns c ON om.organisation_id = c.organisation_id
+            JOIN applications a ON a.campaign_id = c.id
+            JOIN application_ratings ar ON a.campaign_id = c.id
```

Comment:
This last join isn't needed for the join on `application_id`, but is needed later for join on `rating_id`